src/file-token-group/index.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import useBaseComponent from '../internal/hooks/use-base-component';
+import { applyDisplayName } from '../internal/utils/apply-display-name';
+import { FileTokenGroupProps } from './interfaces';
+import InternalFileTokenGroup from './internal';
+
+export { FileTokenGroupProps };
+
+const FileTokenGroup = ({ showFileLastModified, showFileSize, showFileThumbnail, ...props }: FileTokenGroupProps) => {
+  const baseComponentProps = useBaseComponent('FileTokenGroup', {
+    props: {
+      showFileLastModified,
+      showFileSize,
+      showFileThumbnail,
```

Comment:
Let's also add alignment and limit as props and something like useFilesLoading as metadata.

Reply:
What do you mean by useFilesLoading? Is there an example you can share?